Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -1118,6 +1123,20 @@ bool RenderTheme::paintAttachment(const RenderObject&, const PaintInfo&, const I
     return false;
 }
 
+int RenderTheme::attachmentBaseline(const RenderAttachment& attachment) const
+{
+    AttachmentLayout layout(attachment, AttachmentLayoutStyle::NonSelected);
```

Comment:
Do we end up running the layout code multiple times? Maybe that's not new, but it would nice to avoid.

Reply:
Yes this is how it worked before. It could be useful to try and figure out how to only recompute the attachment layout when needed.